server/db/seed/index.js

```diff
@@ -3,85 +3,44 @@ const { Challenge, Image, Solution, User, Userchallenge } = require('../models')
 const conn = require('../conn');
 const { usersSeed } = require('./user');
 const { challengesSeed } = require('./challenge');
-const { userchallengeSeed } = require('./userChallenge');
+const { userchallengeSeed } = require('./userChallenge'); // may delete later
 const { solutionsSeed } = require('./solution');
 const { images } = require('./image');
+const utils = require('../../puppeteer-utils');
 
-// from associations: imageId
-
-// TODO for js - console.log()
-
-// from associations: userId, challengeId, imageId
-
-// from associations: challengeId
-
-// from associations: challengeId, userchallengeId
-
-const currDir = process.cwd();
-const imagesSeed = images.map(image => {
-  image.data = fs.readFileSync(`${currDir}/dist/images/${image.type}/${image.url}`);
-  return image;
-});
+const seedUser = () => {
```

Comment:
Curious why don't these methods need to be async?

Reply:
@theo333 My understanding is that if we want to have seedUser, seedChallenge, and seedSolution happen one by one, then we may need to do async () =>{}.